plugins/storage/volume/sample/src/main/java/org/apache/cloudstack/storage/datastore/lifecycle/SamplePrimaryDataStoreLifeCycleImpl.java

```diff
@@ -146,4 +146,14 @@ public void enableStoragePool(DataStore store) {
     @Override
     public void disableStoragePool(DataStore store) {
     }
+
+    @Override
+    public void changeStoragePoolScopeToZone(DataStore store, ClusterScope clusterScope, HypervisorType hypervisorType) {
+        return;
```

Comment:
Not sure if these return statement add much value

Reply:
removed the return statement